### ISSUE TITLE
Migration broken on Nextcloud 19

### ISSUE THREAD

**Issue by aszlig** [eid-login/eid-login-nextcloud#3]
The [upgrade test from Nextcloud 19 to Nextcloud 20](https://headcounter.org/hydra/log/qnwv5yrqngbcvhdmdaz4ichmddrbq1vf-vm-test-run-nextcloud-upgrade.drv) uncovers this exception when installing the app on Nextcloud 19 via `occ`:

```
Error: Class 'Doctrine\DBAL\Types\Types' not found in .../apps/eidlogin/lib/Migration/Version1000Date20200911113548.php:44
Stack trace:
#0 .../lib/private/DB/MigrationService.php(478): OCA\EidLogin\Migration\Version1000Date20200911113548->changeSchema(Object(OC\Migration\SimpleOutput), Object(Closure), Array)
#1 .../lib/private/DB/MigrationService.php(414): OC\DB\MigrationService->executeStep('1000Date2020091...', false)
#2 .../lib/private/Installer.php(157): OC\DB\MigrationService->migrate()
#3 .../core/Command/App/Enable.php(122): OC\Installer->installApp('eidlogin', false)
#4 .../core/Command/App/Enable.php(92): OC\Core\Command\App\Enable->enableApp('eidlogin', Array, false, Object(Symfony\Component\Console\Output\ConsoleOutput))
#5 .../3rdparty/symfony/console/Command/Command.php(255): OC\Core\Command\App\Enable->execute(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#6 .../3rdparty/symfony/console/Application.php(1012): Symfony\Component\Console\Command\Command->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#7 .../3rdparty/symfony/console/Application.php(272): Symfony\Component\Console\Application->doRunCommand(Object(OC\Core\Command\App\Enable), Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#8 .../3rdparty/symfony/console/Application.php(148): Symfony\Component\Console\Application->doRun(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#9 .../lib/private/Console/Application.php(215): Symfony\Component\Console\Application->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#10 .../console.php(85): OC\Console\Application->run()
#11 .../occ(11): require_once('...
#12 {main}
```

**Disclaimer:** I'm not using this app, nor do I use Nextcloud 19, but I run integration tests across all Nextcloud apps published on https://apps.nextcloud.com/, so I'm only reporting this for the sake of notifying the upstream project.

**Comment by ecmrauh** [comment]
Thanks for your efforts!

**Comment by 7118path** [comment]
@aszlig Thank you for opening this issue. The problem is fixed in 3536c4b9f9e902875e8cc93c7adf1916f403dbb4

The fix will be included in the next release.
